docs/features/software-catalog/external-integrations.md

```diff
@@ -1031,3 +1031,360 @@ backend.add(catalogModuleCustomDataParser);
 
 backend.start();
 ```
+
+## Incremental Entity Provider
+
+The Incremental Ingestion catalog backend module provides an Incremental Entity Provider that can be used to ingest data from sources using delta mutations, while retaining the orphan prevention mechanism provided by full mutations.
+
+### Why did we create it?
```

Comment:
Look, this is nice text as such, but not sure it belongs here. A long motivations section, and technical implementation details, could go into the README.md of the module itself. For this article, it's probably enough to say something along the lines (paraphrasing) of "If your data source is large and perhaps does not fit into memory but does support pagination, there's a convenient incremental entity provider that lets you efficiently ingest that paginated source." And then how to install it. That's what the reader of this particular article most benefits from.